week-2/interpret/time-format.js

```diff
@@ -22,15 +22,18 @@ console.log(formatTimeDisplay(61));
 // Questions
 
 // a) Run this program with node, identify the line the error is thrown. Explain why this error is occurring. How can you fix this error?
+//  Answer: The error was thrown on line 6, 7, 8, 9 10. The error occurred because variable was declared with const within the local scope. By simply removing the const or by replacing it with let.
```

Comment:
This is incorrect, we should keep the consts in the file. There is no error. However don't worry this question has been removed.